pkg/kubernetes/node/v1alpha1/node.go

```diff
@@ -0,0 +1,125 @@
+package v1alpha1
+
+import "k8s.io/api/core/v1"
+
+const (
+	kubeletReady = "KubeletReady"
+)
+
+// node holds the api's node object
+type node struct {
```

Comment:
nit: `Node` make it exported ? useful in case of returning a single `Node` instead of `NodeList` ?